.github/CONTRIBUTING.md

```diff
@@ -161,7 +178,80 @@ If you are porting features/tools from other codebases, you must give them credi
 
+## Things you can work on
+The following list is non-exhaustive, but should give you a good idea of what the dev team would like to see in Pull Requests.
+
+### Spriting
+<details>
+* Replacements of legacy Bay12 sprites
+* Strain specific designs for Aliens for ones that lack them
+* Alternative Alien sprite sets
+* Icon sheet sorting styled after firearms sheets
+* New cosmetic loadout items, such as additional helmet garb
+* Custom tilesets for maps that don’t have them
+* Map specific props and details
+* Map specific Colonist uniforms and equipment
+* Additional HUD styles
+* Bug fixes and inconsistency fixes
+</details>
+
+### Mapping
+<details>
+* Nightmare inserts
+* Object placement quality of life improvements (such as widening hallways and combat lanes cluttered with props)
+* Extra map detailing (so long as it doesn’t negatively impact performance)
+* Removal of dead-ends or gameplay dead-space on existing maps
+* New maps*
+* Bug fixes and inconsistency fixes
+</details>
+
+**A note on new maps.**
+Entirely new maps are generally considered to be stepping stones into the Development team’s mapping dept. proper. However, making a new map is a months long process that requires dedication and constant communication and oversight from mappers on the Maintainer team. Mapping, like spriting and coding is an acquired skill, and it is highly likely your first map is going to suck. Maps are fluid entities that are never absolutely complete, don’t wed yourself to your initial layout, always be prepared to remap half the project when going in.
+
+
+### Coding
+<details>
+* Quality of life improvements that don’t impact gameplay, but improve it
+* Latency optimizations and improvements
+* Backend system refactors that improve server stability or performance
+* Minor features that don’t impact the overall round loop
+* Anything on the public task-board
+* New Alien strains
+* New Marine and Alien tech options
+* Bay12 legacy feature removal (such as wizard backend, laser eyes, etc)
+* Map specific survivor loadouts
+* Bug fixes and inconsistency fixes
+</details>
+
+## What we don't want
+The following list is non-exhaustive, but should give you a good idea of what the dev team don't want to see in Pull Requests.
+
+### Spriting
+<details>
+* Resprites of recently updated content, such as uniforms, guns, marine armor
+* Donor item adjustments or changes
+* Joke sprites
+* Tacticool equipment and gear. We’re retro-future (or cassette punk if you will).
+</details>
+
+### Mapping
+<details>
+* Nightmare inserts with ridiculous loot or ones that are out of place (don’t put snow on LV, for example)
+* Additional detailing that degrades arena space or hinders gameplay in any sort of way
+</details>
+
+### Coding
+<details>
+* Only direct changes to balance numbers on an PR (damage, recoil, health, armor, movespeed, etc), with nothing else to add to the game (without prior approval)
+* No additional species or races, even Arcturians
+</details>
```

Comment:
change all of these to dashes instead of stars

Reply:
Removed <details> and changed * to -